kratos/includes/lock_object.h

```diff
@@ -56,15 +57,9 @@ class LockObject
 
     /// Move constructor.
     KRATOS_DEPRECATED_MESSAGE("The move constructor is deprecated and will be removed in the future!")
-    LockObject(LockObject&& rOther) noexcept
-#ifdef KRATOS_SMP_OPENMP
-			: mLock(rOther.mLock)
-#endif
+    LockObject(LockObject&& rOther)
```

Comment:
remember that moving locks is not possible!
I am only leaving this here as otherwise some code would not compile => should be refactored separately after this is merged

Reply:
Is this still needed? I fixed the usage in memory pool in #9443

Reply:
removed